mungegithub/mungers/sync/issue-sync.go

```diff
@@ -32,6 +32,9 @@ const (
 	// JenkinsBotName is the name of kubekins bot
 	JenkinsBotName = "k8s-bot"
 	priorityPrefix = "priority/P"
+
+	// PriorityFailingTest represents a failing or flaking test
+	PriorityFailingTest = Priority(2)
 	// PriorityP0 represents Priority P0
 	PriorityP0 = Priority(0)
```

Comment:
All of these PriorityLabels can go too.  Safe delete and make sure they're not used anywhere else